docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '2'
+
+services:
+  ci:
+    build:
+      context: .
+      dockerfile: Dockerfile.upgrade_analyzer
+    command: /opt/ci/bin/upgrade_analyzer --listen
+    env_file:
+      - ./kv.conf
+    volumes:
+      - .:/opt/ci
```

Comment:
I played with `docker-compose` as a part of reviewing this.  Definitely should replace my "cheat sheet", but hey, I'm still learning this stuff. 😄